### ISSUE TITLE
How to get proper 'zone's for FileReader input?

### ISSUE THREAD

**Issue by auroralin0408** [pmlmodelling/pyfvcom#5]
Suppose a region spans two zones, how can I get right lon/lat using `FileReader`function?
 I've read codes about `lonlat_from_utm` function, finding out that 'zone' needs to be parallel to nodes in this case, but `parallel` is set default as `False` and won't set to be`True` even `if len(zone) == npos` .
I came to this question because I got this error, and I thought it was becuse a wrong `zone` input.
<img width="1013" alt="image" src="https://user-images.githubusercontent.com/118411170/206738971-820a5991-32df-40d1-a847-1d5bd6e2ff5e.png">
Any answer will be of great help!

**Comment by marius-dewar** [comment]
Can you try again with the updated dev branch? We have recently updated PyFVCOM to be compatible with most recent version of pyproj as all inputs were getting this error (including single zones, I have never tried pyproj spanning multiple zones). Let me know how you get on.

**Comment by jimc101** [comment]
In PyLag, I switched to using EPSG codes. Might be worth considering for PyFVCOM. See [here](https://github.com/pmlmodelling/pylag/blob/84871ab84e8a4070ab1ad916355ef907f79ce748/pylag/processing/coordinate.py#L147) and [here](https://github.com/pmlmodelling/pylag/blob/84871ab84e8a4070ab1ad916355ef907f79ce748/pylag/processing/coordinate.py#L175).

**Comment by marius-dewar** [comment]
Yeah my other suggestion was going to be trying other proj codes that cover the area of interest if you are interested in larger regions.
